packages/react-components/react-storybook-addon/src/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const ADDON_ID = 'storybook/fluentui-react-addon';
```

Comment:
`/` replaced by `_` to make [Storybook's XSS safeguard happy](https://storybook.js.org/docs/react/writing-stories/args#setting-args-through-the-url).